ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -644,6 +664,11 @@ _mp_palm_detection_generate_anchors (properties_MP_PALM_DETECTION *palm_detectio
 static int
 _setOption_mode (bounding_boxes * bdata, const char *param)
 {
+  if (bdata->mode == YOLOV5_BOUNDING_BOX) {
+    bdata->yolov5_pp.scaled_output = (int) g_ascii_strtoll (param, NULL, 10);
```

Comment:
If param is nullable, the return value of g_ascii_strtoll can be 0 on error condition.  Is it okay if 0 is assigned?

Reply:
Any erroneous value would fallback to 0, the default option.

Reply:
Thank you.